sdks/python/apache_beam/runners/interactive/recording_manager.py

```diff
@@ -256,7 +259,7 @@ def describe(self):
 
     size = sum(
         cache_manager.size('full', s.cache_key) for s in self._streams.values())
-    return {'size': size, 'start': self._start}
+    return {'size': size}
```

Comment:
Because the start time wasn't correct if we only start a background caching job. In that case there wouldn't be a new `Recording` so the start time would be 0. I think this also cleans up the logic a bit (no more `min`-ing over all the start times).